src/NzbDrone.Core/Parser/Model/RemoteMovie.cs

```diff
@@ -12,6 +12,16 @@ public class RemoteMovie
         public Movie Movie { get; set; }
         public MappingResultType MappingResult { get; set; }
 
+        public bool IsRecentMovie()
```

Comment:
IMO this makes much more sense being on the actual movie object.

Reply:
I kinda agree, I just added where Sonarr had it.

Reply:
Ah interesting, I would still move it to the movie itself (and also make it a property). Maybe we need it somewhere else in the future.